spec/tracker_spec.rb

```diff
@@ -75,32 +77,13 @@
         expect(e.message).to eq('There are no more pages to retrieve.')
       end
     end
-
-    it 'reuses the params used to retrieve the first page' do
```

Comment:
issue: don't remove tests

Reply:
This is the old helper function for using `get_next_page` which has been replaced by `get_next_page_helper` after consolidating